templates/settings.py.erb

```diff
@@ -14,7 +14,7 @@
 <% if scope['pulpcore::telemetry'].nil? -%>
 # TELEMETRY = False
 <% else -%>
-TELEMETRY = <%= scope['pulpcore::telemetry'] %>
+TELEMETRY = <%= scope['pulpcore::telemetry'].to_s.capitalize %>
```

Comment:
`puppetlabs-stdlib` also has [`to_python`](https://github.com/puppetlabs/puppetlabs-stdlib/blob/main/lib/puppet/functions/to_python.rb) since 8.0.0, but not sure how we feel about bumping that dep?

@ekohl?

Reply:
I made the same comment. We already raised it in 5e2bfd7e335c63998f4abb8d75647cd9837cc2d0 when we started to use `to_python` in the Debian plugin.